.storybook/main.js

```diff
@@ -27,6 +27,10 @@ module.exports = {
     config.resolve.alias = {
       ...config.resolve.alias,
       '@': path.join(__dirname, '../src'),
+      react: 'preact/compat',
+      'react-dom/test-utils': 'preact/test-utils',
+      'react-dom': 'preact/compat', // Must be below test-utils
+      'react/jsx-runtime': 'preact/jsx-runtime',
```

Comment:
i _think_ you might get the same result as with the jest aliasing here. by using the npm aliasing, anytime time node tries to find `react` it will instead find `@preact/compat`, so you may not need these either 🤞

Reply:
I did try this after your previous suggestion in jest, but unfortunately it didn't work in either the storybook or webpack config.